advanced/Scripts/piholeDebug.sh

```diff
@@ -547,17 +547,24 @@ ping_gateway() {
     ping_ipv4_or_ipv6 "${protocol}"
     # Check if we are using IPv4 or IPv6
     # Find the default gateways using IPv4 or IPv6
-    local gateway
+    local gateway gateway_addr gateway_iface
 
     log_write "${INFO} Default IPv${protocol} gateway(s):"
 
     while IFS= read -r gateway; do
-        log_write "     ${gateway}"
-    done < <(ip -"${protocol}" route | grep default | cut -d ' ' -f 3)
+        log_write "     $(cut -d ' ' -f 3 <<< "${gateway}")%$(cut -d ' ' -f 5 <<< "${gateway}")"
```

Comment:
```suggestion
        log_write "     $(cut -d ' ' -f 3 <<< "${gateway}")@$(cut -d ' ' -f 5 <<< "${gateway}")"
```

I think `@` would look nicer.

```
[i] Default IPv4 gateway(s):
     10.0.1.1%end0

[i] Default IPv4 gateway(s):
     10.0.1.1@end0
```

Reply:
I see how you feel `@` looks nicer, but `%` is standardized in [RFC 4007, Scn. 11](https://datatracker.ietf.org/doc/html/rfc4007#section-11) for exactly this purpose. As this is also the syntax used by many tools (`ping`, `dig`, etc.) I don't see a reason the use a deviating notation here.

Reply:
>  If we use interface names as <zone_id>, those addresses could also be
>   represented as follows:
>
>            fe80::1234%ne0
>            ff02::5678%pvc1.3
>            ff08::9abc%interface10